scripts/lldpshow

```diff
@@ -25,30 +25,65 @@ import re
 import sys
 import xml.etree.ElementTree as ET
 from tabulate import tabulate
+import argparse
+import sonic_device_util
+from swsssdk import ConfigDBConnector, SonicDBConfig
+BACKEND_ASIC_INTERFACE_NAME_PREFIX = 'Ethernet-BP'
 
 class Lldpshow(object):
     def __init__(self):
-        self.lldpraw = None
+        self.lldpraw = []
         self.lldpsum = {}
+        self.lldp_interface = []
+        self.lldp_instance = []
         self.err = None
         ### So far only find Router and Bridge two capabilities in lldpctl, so any other capacility types will be read as Other
         ### if further capability type is supported like WLAN, can just add the tag definition here
         self.ctags = {'Router': 'R', 'Bridge': 'B'}
+        SonicDBConfig.load_sonic_global_db_config()
 
-    def get_info(self):
+        # For multi-npu platforms we will get only front-panel interface to display
+        namespaces = sonic_device_util.get_all_namespaces()
+        per_npu_configdb = {}
+        for instance_num, front_asic_namespaces in enumerate(namespaces['front_ns']):
+            per_npu_configdb[front_asic_namespaces] = ConfigDBConnector(use_unix_socket_path=True, namespace=front_asic_namespaces)
+            per_npu_configdb[front_asic_namespaces].connect()
+            self.lldp_interface.append('')
```

Comment:
You are appending the empty string here to signify the host (i.e., no) namespace, correct? I think you should create a constant for this so that it is more clear, like in the other PRs.

Reply:
@jleveque Done